backend/src/StdLibCliHost/paket.references

```diff
@@ -0,0 +1,3 @@
+Ply
+FSharp.Core
+FSharpPlus
```

Comment:
the FSharpPlus reference may be remove-able - not sure. (guessing the file was copied from the other project verbatim)